go/processor/geth_eth/processor.go

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2024 Fantom Foundation
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file and at fantom.foundation/bsl11.
+//
+// Change Date: 2028-4-16
+//
+// On the date above, in accordance with the Business Source License, use of
+// this software will be governed by the GNU Lesser General Public License v3.
+
+package geth_processor_eth
+
+import (
+	geth_processor "github.com/0xsoniclabs/tosca/go/processor/geth"
+	"github.com/0xsoniclabs/tosca/go/tosca"
+)
+
+func init() {
+	tosca.RegisterProcessorFactory("geth-eth", ethereumProcessor)
```

Comment:
Please add a line of documentation describing the properties of the processor that is registered under the name `geth-eth`.